airflow/providers/microsoft/azure/triggers/data_factory.py

```diff
@@ -89,3 +90,106 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:
                 await asyncio.sleep(self.poke_interval)
         except Exception as e:
             yield TriggerEvent({"status": "error", "message": str(e)})
+
+
+class AzureDataFactoryTrigger(BaseTrigger):
+    """
+    AzureDataFactoryTrigger is triggered when Azure data factory pipeline job succeeded or failed.
+    When wait_for_termination is set to False it triggered immediately with success status
+    :param run_id: Run id of a Azure data pipeline run job.
```

Comment:
```suggestion
    When wait_for_termination is set to False it triggered immediately with success status

    :param run_id: Run id of a Azure data pipeline run job.
```
When Triggers eventually make it to the Python API docs, this empty line is required for the parameters to render correctly.